timescaledb/how-to-guides/backup-and-restore/pg-dump-and-restore.md

```diff
@@ -114,6 +118,8 @@ partitions, or the chunk interval sizes.
 
 </procedure>
 
+## Troubleshooting
+
```

Comment:
There's a standard sentence we usually put in here. Check one of the other troubleshooting sections to find it.

Reply:
```suggestion

This section contains some ideas for troubleshooting common problems experienced with backups and restores.

```